src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -255,7 +255,10 @@ public bool ShowDebugASTs
 
         public bool WillForceReExecuteOfNode
         {
-            get { return NodeModel.ForceReExecuteOfNode; }
+            get
+            {
+                return NodeModel.ExecutionHint.HasFlag(NodeModel.ExecutionHints.ForceExecute);
+            }
```

Comment:
Please consider reusing `NodeModel.NeedsForceExecution` flag (as suggested in overall comments below).